bindings/python/test.py

```diff
@@ -57,6 +57,53 @@ def test_torrent_handle(self):
 		self.h.prioritize_pieces([(0, 1)])
 		self.assertEqual(self.h.piece_priorities(), [1])
 
+	def test_unknown_torrent_parameter(self):
+		self.ses = lt.session({'alert_mask': lt.alert.category_t.all_categories,
+			'enable_dht': False})
+		try:
+			self.h = self.ses.add_torrent({'storage': ''})
+			self.assertFalse('should have thrown an exception')
+		except KeyError as e:
+			print(e)
+
+	def test_torrent_parameter(self):
+		self.ses = lt.session({'alert_mask': lt.alert.category_t.all_categories,
+			'enable_dht': False})
+		self.ti = lt.torrent_info('url_seed_multi.torrent');
+		self.h = self.ses.add_torrent({
+			'ti': self.ti,
+			'save_path': os.getcwd(),
+			# TODO: Expecting an object of type list; got an object of type str instead
+			# I don't know what is wrong at the binding
+			#'trackers': ['http://test.com/announce'],
```

Comment:
your comment here suggests that "trackers" is expected to be a list of list of strings. which kind of makes sense. On the C++ side, the ``tracker_tiers`` is used to indicate which "list" each torrent belongs to, and ``trackers`` is flat instead. We should make sure though that ``load_resume_data`` is compatible with this